python/triton/runtime/__init__.py

```diff
@@ -21,3 +23,9 @@
     "RemoteCacheBackend",
     "TensorWrapper",
 ]
+
+# TritonParse initialization must be called after knobs initialization
+if importlib.util.find_spec("tritonparse") is not None:
+    import tritonparse.structured_logging
+
+    tritonparse.structured_logging.init_with_env()
```

Comment:
it feels weird to import tritonparse explicitly while triton doesn't depend on it. 
Can this be done on user or tritonparse side?